index.html

```diff
@@ -12,113 +12,107 @@
     <!-- Apple Icon -->
     <link rel="apple-touch-icon" sizes="152x152" href="./img/apple-touch-icon.png">
     <!-- Styles -->
-    <link rel="stylesheet" href="./csh-material-bootstrap/dist/css/csh-material-bootstrap.css">
+    <link rel="stylesheet" href="https://s3.csh.rit.edu/csh-material-bootstrap/4.0.0/dist/csh-material-bootstrap.min.css" media="screen">
     <link rel="stylesheet" href="./css/members.css">
     <!-- Scripts -->
     <script src="./vendor/js/jquery-2.1.3.min.js"></script>
-    <script src="./vendor/js/bootstrap.min.js"></script>
+    <!--<script src="./vendor/js/bootstrap.min.js"></script>-->
```

Comment:
Let's just remove this if we're no longer using it.

Reply:
Whoops. I thought I had done this. Will probably commit this tommorow.

Reply:
If we remove it, also remove it, and any other unused libraries from the package.json